### PR TITLE
[ᚬmaster] feat: add circulating supply api

### PR DESCRIPTION
**Pull request by shaojunda**

### CODE REVIEW

**Comment by shaojunda** [comment]
https://github.com/cryptape/ckb-explorer-internal/issues/364

File: app/models/market_data.rb [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  end
+
+  def total_supply
+    result = parsed_dao.c_i - BURN_QUOTA
+
+    (result / 10**8).truncate(8)
+  end
+
+  def circulating_supply
+    result = parsed_dao.c_i - BURN_QUOTA - ecosystem_locked - team_locked - private_sale_locked - founding_partners_locked - foundation_reserve_locked
```
Comment (doitian) [inline app/models/market_data.rb]: I think unissued secondary rewards (`parsed_dao.S_i`) should be excluded from the circulating supply.

File: app/models/market_data.rb [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+    (result / 10**8).truncate(8)
+  end
+
+  def ecosystem_locked
+    first_released_time = Time.zone.parse("2020-07-01")
```
Comment (doitian) [inline app/models/market_data.rb]: Use UTC timezone explicitly instead of `Time.zone`

Comment (shaojunda) [reply app/models/market_data.rb]: thanks, I will fix this.

File: app/models/market_data.rb (hunk 1 above)
Comment (shaojunda) [reply app/models/market_data.rb]: updated: https://github.com/nervosnetwork/ckb-explorer/pull/487/commits/b003bdb3df0ed7496323b78e00e0d4b324a0018e

**Pull request merged**